static/src/js/components/Map/__tests__/ShapefileLayerExtended.test.js

```diff
@@ -0,0 +1,121 @@
+import { ShapefileLayerExtended } from '../ShapefileLayerExtended'
+
+describe('ShapefileLayerExtended', () => {
+  describe('separateMultiPolygons', () => {
+    let layer
+
+    beforeEach(() => {
+      layer = new ShapefileLayerExtended({})
+    })
+
+    it('should separate MultiPolygon features into individual Polygon features', () => {
+      const geojson = {
```

Comment:
Just a nitpick so someone else may disagree but, having `const layer = new ShapefileLayerExtended({}) ` in each test seems cleaner than doing the object creation from the beforeEach block

Reply:
Agreed, implemented in latest commit